lib/SYCL/inSPIRation.cpp

```diff
@@ -189,12 +190,17 @@ struct inSPIRation : public ModulePass {
     F.setMetadata("kernel_arg_addr_space",
                   llvm::MDNode::get(Ctx, AddressSpaceQuals));
 
-    // Add the SPIR metadata describing the type of each argument
-    F.setMetadata("kernel_arg_type", llvm::MDNode::get(Ctx, Types));
+    // Now, since kernel_arg_type and kernel_arg_base_type with types
+    // that are not c++ fundamental types would choke xocc, we remove
```

Comment:
`that are not c++ fundamental types would choke xocc, we remove` → `that are not C++ fundamental types  choke SPIR consumers such as xocc, we avoid`